tests/docker/start.irods.ubuntu22.sh

```diff
@@ -17,7 +22,15 @@ echo Postgres took approximately $counter seconds to fully start ...
 /var/lib/irods/scripts/setup_irods.py < /var/lib/irods/packaging/localhost_setup_postgres.input
 
 #### Start iRODS ####
-service irods start
+sudo -H -u irods -g irods bash -c "irodsServer -d"
```

Comment:
Is there a reason to not continue using `service irods start` here?

Reply:
It wasn't working.

Reply:
Hmm... okay. If it works, let's do it. Resolving.